controls/radrichtextbox/features/mail-merge/insert-new-line-in-merge-field.md

```diff
@@ -0,0 +1,158 @@
+slug: radrichtextbox-features-mail-merge-insert-new-line-in-merge-field
+tags: mail,merge,vertical,tab,line
+published: True
+position: 1
+---
+
+# Insert a New Line in Merge Field
+
+In RadRichtextBox, you can insert a line break using the [Break element]({%slug radrichtextbox-features-document-elements-break%}). However, when working with mail merge fields, inserting a similar document element is not possible. This is why we introduced support for the vertical tab symbol. This article will explain how you could use this symbol in order to insert a new line in the document content and in a merge field. 
+
+* [What is the Vertical Tab Symbol?](#what-is-the-vertical-tab-symbol)
+
+* [How to Insert a Vertical Tab in RadDocument?](#how-to-insert-a-vertical-tab-in-raddocument)
+
+* [Insert a Line Break Before/After a Merge Field If the Field Result Isn't Empty](#insert-a-line-break-beforeafter-a-merge-field-if-the-field-result-isnt-empty)
+
+* [Using Vertical Tab in a DocumentVariableField](#using-vertical-tab-in-a-documentvariablefield)
+
+
+## What is the Vertical Tab Symbol?
+
+The vertical tab character is __*"\v"*__. It can be inserted directly into a document as a string or through the merge field switches. The vertical tab character itself won’t be preserved, but it is evaluated to a [Break element]({%slug radrichtextbox-features-document-elements-break%}) of type LineBreak.
+
+## How to Insert a Vertical Tab in RadDocument?
+
+
+The [RadDocumentEditor class]({%slug radrichtextbox-features-raddocumenteditor%}) exposes the Insert() method that allows you add text to the document. **Example 1** shows how to insert a vertical tab with this method.
+
+#### __[C#] Example 1: Insert vertical tab through RadDocumentEditor__
+
+{{region radrichtextbox-features-mail-merge-Insert-new-line-in-merge-field_0}}
+
+	radDocumentEditor.Insert("\v");
+{{endregion}}
+
+#### __[VB.NET] Example 1: Insert vertical tab through RadDocumentEditor__
+
+{{region radrichtextbox-features-mail-merge-Insert-new-line-in-merge-field_1}}
+
+	radDocumentEditor.Insert("\v")
+{{endregion}}
+
+
+Another option is to create a [Span]({%slug radrichtextbox-features-document-elements-span%}) and assign the vertical tab character to it. 
+
+#### __[C#] Example 2: Create vertical tab through document model__
+
+{{region radrichtextbox-features-mail-merge-Insert-new-line-in-merge-field_2}}
+
+	Span span = new Span("\v");
+{{endregion}}
+
+
+#### __[VB.NET] Example 2: Create vertical tab through document model__
+
+{{region radrichtextbox-features-mail-merge-Insert-new-line-in-merge-field_3}}
+
+	Dim span As New Span("\v")
+{{endregion}}
+When inserted in the document, the vertical tab symbol will be replaced with a Break element of type LineBreak.
+
+## Insert a Line Break Before/After a Merge Field If the Field Result Isn't Empty
+
+You can use the vertical tab symbol when you need to insert a line break before or after the result of a merge field only if it is not empty.
```

Comment:
Can we make this a more extensive explanation of the scenario? Just to make sure clients hit it when searching for similar questions.